docs/source/JavaScriptUsage.md

```diff
@@ -16,43 +16,28 @@ documentation to build `flatc` and should be familiar with
 
 ## FlatBuffers JavaScript library code location
 
-The code for the FlatBuffers JavaScript library can be found at
-`flatbuffers/js`. You can browse the library code on the [FlatBuffers
-GitHub page](https://github.com/google/flatbuffers/tree/master/js).
-
-## Testing the FlatBuffers JavaScript library
-
-The code to test the JavaScript library can be found at `flatbuffers/tests`.
-The test code itself is located in [JavaScriptTest.js](https://github.com/
-google/flatbuffers/blob/master/tests/JavaScriptTest.js).
-
-To run the tests, use the [JavaScriptTest.sh](https://github.com/google/
-flatbuffers/blob/master/tests/JavaScriptTest.sh) shell script.
-
-*Note: The JavaScript test file requires [Node.js](https://nodejs.org/en/).*
+The generated code for the FlatBuffers JavaScript library can be found at 
+https://www.npmjs.com/package/flatbuffers. To generate the library from sources,
+check out https://github.com/google/flatbuffers/wiki/FlatBuffers-release-process.
```

Comment:
That's not a very friendly way to tell a JS programmer what to do? that page is filled with cruft not relevant to a JS programmer. Can we have more baby-steps style explanation?

Again, maybe I am missing something since I am not familiar with the eco system, but I can imagine JS programmers needing to build things themselves when they're working with JS from the repo rather than a released version, right?

Reply:
I'll take the approach of using it as a normal dependency, instead of generating anything.